packages/volto/src/server.jsx

```diff
@@ -224,9 +224,33 @@ server.get('/*', (req, res) => {
 
   loadOnServer({ store, location, routes, api })
     .then(() => {
+      const defaultLanguage =
+        store.getState().site?.data['plone.default_language'] || 'en';
+
+      const supportedLanguages =
+        store.getState().site?.data['plone.available_languages'] || 'en';
+
+      // Load the lang files when they matter
+      // We have to investigate if the L46 block is really necessary then
```

Comment:
This is a good question. At a first glance I'd like to remove L46 but it looks like more refactoring would be needed. On the other hand, without removing that part, the import part from this new code looks useless.